ci/codecov.sh

```diff
@@ -23,51 +23,69 @@ set -ex
 if [[ "$1" == "setup" ]]; then
     export B2_VARIANT=debug
     if [ -z "$B2_CI_VERSION" ]; then
-        export B2_CXXFLAGS="${B2_CXXFLAGS:+$B2_CXXFLAGS }cxxflags=-fkeep-static-functions cxxflags=--coverage"
+        export B2_CXXFLAGS="${B2_CXXFLAGS:+$B2_CXXFLAGS }cxxflags=-fprofile-arcs cxxflags=-ftest-coverage cxxflags=--coverage"
```

Comment:
`--coverage` is already added and a synonym for `-fprofile-arcs -ftest-coverage` according to e.g. https://stackoverflow.com/questions/28840261

Also it removes a switch meant to avoid dropping covered lines due to possible inlining etc.
So why did you change this? "Fixed coverage cxxflags." doesn't explain this as it looks like actually worsening stuff. Or was anything wrong with it?

Reply:
It did not work with clang, I was trying to get clang to work, that's why I removed it.  I will put it back.